repos/system_upgrade/common/actors/setetcreleasever/actor.py

```diff
@@ -0,0 +1,29 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import setetcreleasever
+from leapp.models import PackageManagerInfo
+from leapp.tags import FirstBootPhaseTag, IPUWorkflowTag
+
+
+class SetEtcReleaseVer(Actor):
+    """
+    Set target release version in cases where the "/etc/{yum,dnf}/var/releasever" file exists
+
+    If the file exists, whatever is in there, is attached into repository URL. In case the file is
+    empty, an empty string is attached to the URL causing it to be invalid, also when it is set to
+    the source version (e.g 7.7), the URL on the target system is again invalid. In order to avoid
+    such scenario, we popullate the file with the target system version whenever it exists.
+
+    """
+
+    name = 'set_etc_releasever'
+    consumes = (PackageManagerInfo,)
+    produces = ()
+    tags = (IPUWorkflowTag, FirstBootPhaseTag)
```

Comment:
These actions should be done during the application phase `ApplicationsPhaseTag`. There is no reason why it should happen so late (we do not need an internet connection to do it). The firstbootphase should be used only by actors that needs a booted target OS (or internet connection nowadays).

```suggestion
    tags = (IPUWorkflowTag, ApplicationsPhaseTag)
```

Reply:
Fixed.